src/bus/webSocketConnection.spec.js

```diff
@@ -36,6 +39,335 @@ describe('WebScoketConnection', function() {
     });
   });
 
+  describe('authorize', function() {
+    context('on a successful message', function() {
+      context('when a user is authorized', function() {
+        let expectedMessage;
+        let expectedOrganization;
+        let expectedJsonRpc;
+        let jsonRpcId;
+        let promise;
+        let send;
+        let token;
+        let ws;
+
+        beforeEach(function() {
+          expectedMessage = { jsonrpc: '2.0', id: '0', result: null };
+          expectedOrganization = fixture.build('organization');
+          send = this.sandbox.stub(expectedWebSocket, 'send');
```

Comment:
I don't think you need to stub this -- we can utilize the features of mock-socket more. I don't think mock-socket does as much as I think sometimes, but could just spy here